src/main/java/com/epam/ta/reportportal/core/launch/attribute/impl/RetentionPolicyAttributeHandler.java

```diff
@@ -93,8 +108,16 @@ public void handleLaunchUpdate(Launch launch) {
 
       if ("important".equalsIgnoreCase(retentionPolicyNewAttribute.getValue())) {
         launch.setRetentionPolicy(RetentionPolicyEnum.IMPORTANT);
+        messageBus.publishActivity(
+            new MarkLaunchAsImportantEvent(TO_ACTIVITY_RESOURCE.apply(launch), user.getUserId(),
+                user.getUsername()
+            ));
       } else if ("regular".equalsIgnoreCase(retentionPolicyNewAttribute.getValue())) {
         launch.setRetentionPolicy(RetentionPolicyEnum.REGULAR);
+        messageBus.publishActivity(
```

Comment:
duplicated `messageBus.publishActivity(...)` could be called once after `if-else` block